glfs.c

```diff
@@ -430,58 +430,6 @@ static char* tcmu_get_path( struct tcmu_device *dev)
 	return config;
 }
 
-
-static bool glfs_check_config(const char *cfgstring, char **reason)
-{
```

Comment:
Apologies for delay.

Mike, Yes I did not understand the intention of check_config may be because it is already broken.

Thanks for your very detailed input, as now it seems to make sense about netlink interface implementation for me.

IMO, it is very helpful to document some info about check_config/dbus/netlink intention somewhere in this repo.

> I started working on the nl fix. I will post the nl fix patches in another PR when I am done.

Cool :-)

> Go ahead and assume I am going to finish the nl patches around the same time you repush and just include the glfs check_config removal. I will handle the other ones in my PR.

I'm curious to cross check my understanding here, are you fixing the nl functionality using .open() now, rather it was with .check_config() before ?